src/app/component/layout/components/footer/footer.component.html

```diff
@@ -1,78 +1,60 @@
 <footer>
-  <div class="main-background-image">
-    <div class="main">
-      <a [routerLink]="['/']"
-         href=""
-         class="small-screen-size">
-        <img [src]="footerImageList.greenCityLogo"
-             class="logo"
-             alt="GreenCity Logo">
-      </a>
-      <div class="links">
-        <div class="left-side">
-          <ul class="list-of-site-links">
-            <li>
-              <a [routerLink]="['/news']"
-                 class="router-links"
-                 href="">{{ "user.footer.eco-events" | translate }}
-              </a>
-            </li>
-            <li>
-              <a href=""
-                 class="router-links">{{ "user.footer.tips" | translate }}
-              </a>
-            </li>
-            <li>
-              <a [routerLink]="['/map']"
-                 class="router-links"
-                 href="">{{ "user.footer.map" | translate }}
-              </a>
-            </li>
-            <li>
-              <a [routerLink]="['/about']"
-                 class="router-links"
-                 href="">{{ "user.footer.about-us" | translate }}
-              </a>
-            </li>
-            <li>
-              <a [routerLink]="['/profile', getUserId()]"
-                 href=""
-                 class="router-links">{{ "user.footer.my-habits" | translate }}
-              </a>
-            </li>
-          </ul>
+  <div class="footer_background-image">
```

Comment:
Why did you use BEM methodology? According to project's style guide we should use dashes in class names, isn't it?

Reply:
it's the same for me.
just BEM expands it..